framework/src/modules/chain/transport.js

```diff
@@ -108,7 +108,9 @@ class Transport {
 	 */
 	// eslint-disable-next-line class-methods-use-this
 	onSignature(signature, broadcast) {
-		if (broadcast && !__private.broadcaster.maxRelays(signature)) {
+		if (broadcast) {
+			// TODO: Remove the relays property as part of the next hard fork. This needs to be set to a fixed value for backwards compatibility.
+			signature.relays = 1;
```

Comment:
If we are going to backward compatible way, I think we should keep track of release limit, otherwise with previous implementation, it will propagate too much